packages/suite-data/files/translations/en.json

```diff
@@ -1740,6 +1740,7 @@
   "TR_STAKE_UNSTAKED_AND_READY_TO_CLAIM": "Unstaked and ready to claim",
   "TR_STAKE_UNSTAKE_TO_CLAIM": "Unstake to claim",
   "TR_STAKE_UNSTAKING": "Unstaking",
+  "TR_STAKE_UNSTAKING_APPROXIMATE": "Approximate ETH available instantly",
```

Comment:
do not modify `en.json` please, just `messages.json`

Reply:
fixed